ui/src/app/landing/landing.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="landingPaymentOptions">
+  <h4>Please select a payment type</h4>
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <button type="button" routerLink="/creditcard" routerLinkActive="active" class="btn btn-outline-secondary">Credit Card</button>
+      </div>
+    </div>
+    <div class="w-100"></div>
```

Comment:
Full column to force a new row? Why not use a `.row` with columns as intended?
```html
<div class="form-group row">
  <div class="col">
      LABEL
  </div>
   <div class="col">
     FIELD
  </div>
</div>
<div class="form-group row">
  <div class="col">
      LABEL
  </div>
   <div class="col">
     FIELD
  </div>
</div>
```

Reply:
again design session